lib/modules/platform/azure/azure-helper.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('modules/platform/azure/azure-helper', () => {
       const mockEventStream = new Readable({
         objectMode: true,
 
-        read: function () {
+        read() {
```

Comment:
If this is not wanted it can be ignored by setting the option methods to false

Reply:
I'm okay with it.